### DISCUSSION TITLE
Soketi not working with Laravel and Nginx

### DISCUSSION THREAD

**Discussion by 60poro** [soketi/pws#377] [General]
I wanted to try to use the soketi from Laravel and Nginx yesterday. I did everything according to official documents, but it didn't work for me. I was getting a message that the app key is invalid. Can I have a working example please? With the nginx configuration and everything?

**Comment by ddahan** [comment]
@60poro I had the same issue with Python lib. The issue was due to the pusher lib that expects the app id to use digits only (while the default Soketi example uses letters, which is confusing).
See: https://github.com/soketi/soketi/issues/491

**Comment by angeljqv** [comment]
Everything is working
